src/ImagineSvg/Imagine.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\ImagineSvg;
+
+use Imagine\Image\AbstractImagine;
+use Imagine\Image\Metadata\MetadataBag;
+use Imagine\Image\Palette\Color\ColorInterface;
+use Imagine\Image\BoxInterface;
+use Imagine\Exception\InvalidArgumentException;
+use Imagine\Exception\RuntimeException;
+
+/**
+ * Imagine implementation for SVG images
+ *
+ * @author Martin Auswöger <martin@auswoeger.com>
+ */
+class Imagine extends AbstractImagine
+{
+    /**
+     * @throws RuntimeException
+     */
+    public function __construct()
+    {
+        if (!extension_loaded('dom')) {
```

Comment:
Then it should be a `suggest` :-) But it is a requirement for Contao 3 as well, so why would we drop that now? :)